src/style.css

```
@@ -1 +1,127 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 900;
+  font-style: normal;
+  font-display: swap;
+}
+
+/* FONTS */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --color-white: #fff;
+}
+
+.app {
+  font-weight: 300;
+}
+
+.app__wrapper {
+  padding: 20px 8.5px 20px 8px;
```

```suggestion
  padding: 20px 8px 20px 8px;
```
It's better to use integers without a dot

---

Yes ! I 100% agree with you but I could not pass tests otherwise